README.md

```diff
@@ -695,8 +770,23 @@ Private properties should be declared in class extensions (anonymous categories)
 @end
 ```
 
+## Private headers
+
+Use private headers to define class extension and any internal methods and properties which can be used for [unit tests](#unit-tests). Also they have to contain all necessary inclusions.
```

Comment:
So, private headers should contain:
 - imports of headers for types that was declared in main header using "forward declaration"
 - imports of headers for types that was used in private header

Private header should not contain:
 - imports of headers for types that was not declared in main or private headers but was used in implementation file